src/Integration/NewConnectedMode/ConfigurationSerializer.cs

```diff
@@ -96,13 +78,14 @@ internal static string GetConnectionFilePath(string solutionFilePath)
             return Path.Combine(solutionFolder, ".sonarlint", $"{solutionName}.sqconfig");
         }
 
-        private string GetCurrentSolutionFilePath()
+        protected override string GetFullConfigurationFilePath()
```

Comment:
The only thing this class has to do is calculate the full path for the config file.